pkg/cmd/container/run_mount.go

```diff
@@ -307,6 +308,9 @@ func generateMountOpts(ctx context.Context, client *containerd.Client, ensuredIm
 	for _, c := range containers {
 		ls, err := c.Labels(ctx)
 		if err != nil {
+			if errors.Is(err, errdefs.ErrNotFound) {
```

Comment:
Can we print a warning

Reply:
Mmm... I am on the fence.

It does not feel like an abnormal circumstance to me. It just means that a container that we retrieved from `client.Containers` a moment ago has been removed since.
Could happen naturally because a container was started with `--rm` and just exited - or because of concurrent operations (like we see on the CI).

Let me know your thoughts ^.

Reply:
What about Debugf?

Reply:
Sure.

Reply:
Debugf added (and also in the other place I fixed earlier) + a comment.